src/app/Topology/Shared/Entity/EntityDetails.tsx

```diff
@@ -213,7 +213,7 @@ export const TargetDetails: React.FC<{
           'Target',
           ['labels'],
         ),
-        content: <EntityLabels labels={serviceRef.labels} maxDisplay={3} />,
+        content: <EntityKeyValues kv={serviceRef.labels} maxDisplay={3} />,
```

Comment:
I think the ones with labels should be passed a transform function (i.e. ` keyValueEntryTransformer`) while ones with classpath can pass as undefined.